.github/ISSUE_TEMPLATE/als-----wil-ik----.md

```diff
@@ -8,24 +8,28 @@ about: Beschrijving van een feature/user story
 
 **Definition of ready**
 - [ ] Iedereen in het team begrijpt de user story
+- [ ] de gewenste (aanvulling op de) functionaliteit van de API's duidelijk en beschreven is.
 - [ ] Is klein genoeg (maximaal 1/5 van sprint)
-- [ ] Product Owner akkoord
+- [ ] Product Owner akkoord en voorzien van prioriteit (mag alleen afgevinkt worden door Eelco)
 - [ ] Voorzien van acceptatiecriteria (duidelijk en testbaar)
 - [ ] Voorzien van Definition of Done (duidelijk en testbaar)
 - [ ] Voorzien van taken
 - [ ] Idee hebben van hoe deze user story kan worden gedemonstreerd.
 - [ ] _**Userstory is ingevuld op template architectuurschets**_
-- [ ] **_Userstory is voorzien van veldmapping naar RGBZ2_**
-- [ ] **_Userstory past op wenselijk gebruik ZGW volgens GEMMA 2_**
+- [ ] **_Userstory is voorzien van veldmapping naar RGBZ2_** (mag alleen afgevinkt worden door Arjan)
+- [ ] **_Userstory past op wenselijk gebruik ZGW volgens GEMMA 2_** (mag alleen afgevinkt worden door Jeffrey)
 - [ ] Vastgelegd in Github en geplaatst in kolom ready
 
 **Definition of done**
 - [ ] er is een OAS 3.0 specificatie
-- [ ] de specificatie is gepubliceerd leesbaar
+- [ ] de specificatie is gepubliceerd leesbaar (functioneel en technisch) 
```

Comment:
Wat wordt bedoeld met ' de specificatie'? In de regel er boven staat de ' OAS 3.0 specifiatie'. Wordt er dan gedoeld op die OAS-specificatie? De functionele documentatie waar we het nu al weken over hebben, maakt daar geen deel van uit. Waar is dan de functionele documentatie?